universal-login-react/src/ui/styles/topUpDefaults.sass

```diff
@@ -36,6 +36,14 @@
       font-weight: 600
       text-transform: uppercase
 
+    .qr-code-wrapper
+      text-align: center
+      border-radius: 6px
+      border: none
+      background: #362F6A
+      margin: 15px 0px
+      padding: 15px 0px
+
```

Comment:
Remove layout styles(margin, padding) from here. In topUpDefaults should be only styles associated with appearance.

Reply:
✔️